src/inc/msquic.h

```diff
@@ -923,6 +923,7 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #define QUIC_PARAM_CONN_STATISTICS_V2                   0x05000016  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_STATISTICS_V2_PLAT              0x05000017  // QUIC_STATISTICS_V2
 #define QUIC_PARAM_CONN_ORIG_DEST_CID                   0x05000018  // uint8_t[]
+#define QUIC_PARAM_CONN_DSCP                            0x50000019  // uint8_t
```

Comment:
Maybe call it `_SEND_DSCP`?
```suggestion
#define QUIC_PARAM_CONN_SEND_DSCP                       0x50000019  // uint8_t
```

Reply:
I was thinking of the same pattern that socket options use where the send one has no prefix, but the receive one has a prefix. So when we do the last-received DSCP value, it would be `QUIC_PARAM_CONN_RECV_DSCP`